src/js/actions/layers.js

```diff
@@ -1377,45 +1382,147 @@ define(function (require, exports) {
     };
 
     /**
+     * Helper function to find the right most artboard 
+     * @param {Immutable.List.of<Layer>} artboards
```

Comment:
Just the constructor should be used in the type expression, so this should be `Immutable.List.<Layer>`.